ldk/javascript/src/webpack/config.ts

```diff
@@ -2,9 +2,9 @@ import * as webpack from 'webpack';
 import * as path from 'path';
 import Terser from 'terser-webpack-plugin';
 import { generateBanner } from './generate-banner';
-import { LdkSettings } from './ldk-settings';
+import { LdkSettings } from './ldk-settings'; // Need to dynamically refer to Loop's package.json
 
-/* eslint-disable-next-line */ // Need to dynamically refer to Loop's package.json
```

Comment:
I realize that the bulk of this PR is prettier doing its thing. Is it worth keeping why we need to disable the linter? Maybe change the eslint disable to reference the exact rule we are breaking

Reply:
Ooo yes, great point, I try to document the specific rule and why I'm suppressing it, and I should do that here too.

Reply:
Is done!